llvm/lib/CodeGen/AsmPrinter/AccelTable.cpp

```diff
@@ -38,7 +38,9 @@ void AccelTableBase::computeBucketCount() {
   for (const auto &E : Entries)
     Uniques.push_back(E.second.HashValue);
 
-  BucketCount = llvm::dwarf::getDebugNamesBucketCount(Uniques, UniqueHashCount);
+  auto Counts = llvm::dwarf::getDebugNamesBucketAndHashCount(Uniques);
```

Comment:
This can be simplified to `std::tie(BucketCount, UniqueHashCount)`

Reply:
+1 to this - @cmtice perhaps you could make this change as a follow-up commit. No need to send it for review. (but please follow up on this comment thread and mention the commit hash of the change once it's upstream)

Reply:
I made the simplification already. Sometimes reviewers may click "Accept" and expect some minor code suggestions will be adopted before landing. It's worth paying attention to these comments.

(For both #82153 and this #83047, my comments were created before the patch landed. Don't worry. We all made mistakes.)

Reply:
ah, https://github.com/llvm/llvm-project/commit/7b0b64a7016d67d6d8c538fe406a2ffd93c11602 cool, thanks!